easybuild/easyconfigs/s/Salmon/Salmon-0.12.0-foss-2018b.eb

```diff
@@ -15,15 +15,23 @@ source_urls = [GITHUB_LOWER_SOURCE]
 sources = ['v%(version)s.tar.gz']
 checksums = ['91ebd1efc5b0b4c12ec6babecf3c0b79f7102e42b8895ca07c8c8fea869fefa3']
 
-builddependencies = [('CMake', '3.12.1')]
-
-runtest = 'test'
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('pkg-config', '0.29.2'),
+    ('jemalloc', '5.1.0'),
+]
 
 dependencies = [
     ('Boost', '1.67.0'),
     ('tbb', '2018_U5'),
 ]
 
+configopts = '-DJEMALLOC_ROOT=$EBROOTJEMALLOC'
+
+# need to unset the $LD_PRELOAD set be jemalloc to avoid hanging 'make test'
```

Comment:
please add that it's statically linked in

Reply:
done in 4e085fbabf